src/main/java/ch/cern/eam/wshub/core/services/workorders/ChecklistService.java

```diff
@@ -21,10 +21,11 @@
     @Operation(logOperation = INFOR_OPERATION.WO_CHECKL_FOLLOWUPWO_C)
     Long createFollowUpWorkOrders(InforContext context, Activity activity) throws InforException;
 
-    //TODO: FIX @OPERATION
-    //    @Operation(logOperation = INFOR_OPERATION.WO_CHECKL_U)
+    @Operation(logOperation = INFOR_OPERATION.WO_CHECKL_ESGN_U)
     WorkOrderActivityChecklistSignatureResponse eSignWorkOrderActivityChecklist(InforContext context, WorkOrderActivityCheckListSignature workOrderActivityCheckListSignature)
-            throws Exception;
+            throws InforException;
 
-    WorkOrderActivityChecklistSignatureResult[] getSignatures(InforContext context, String workOrderCode, String activityCode) throws Exception;
+    @Operation(logOperation = INFOR_OPERATION.WO_CHECKL_ESGN_R)
+    WorkOrderActivityChecklistSignatureResult[] getSignatures(InforContext context, String workOrderCode, String activityCode)
+            throws  InforException;
```

Comment:
2 spaces between throws and InforException.

Reply:
Fixed.